src/stacks-hierarchy/Root_FareContractDetailsScreen.tsx

```diff
@@ -1,29 +1,43 @@
 import {FullScreenHeader, useTicketInfo} from '@atb/components/screen-header';
-import {DetailsContent} from '@atb/fare-contracts';
+import {CarnetDetailedView, DetailsContent} from '@atb/fare-contracts';
 import {useApplePassPresentationSuppression} from '@atb/suppress-pass-presentation';
 import {StyleSheet} from '@atb/theme';
-import {
-  FareContractTexts,
-  useTranslation,
-} from '@atb/translations';
+import {FareContractTexts, useTranslation} from '@atb/translations';
 import React from 'react';
 import {ScrollView, View} from 'react-native';
 import {RootStackScreenProps} from '../stacks-hierarchy/navigation-types';
 import {useTimeContextState} from '@atb/time';
 import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {useAnalytics} from '@atb/analytics';
 
 type Props = RootStackScreenProps<'Root_FareContractDetailsScreen'>;
 
+export type TicketType = 'normal' | 'carnet';
+
 export function Root_FareContractDetailsScreen({navigation, route}: Props) {
   const styles = useStyles();
   const {t} = useTranslation();
   const {enable_ticket_information} = useRemoteConfig();
   const {serverNow} = useTimeContextState();
-  const {navigateToTicketInfoScreen, fareContract, preassignedFareProduct} =
-    useTicketInfo(route?.params?.orderId);
+  const analytics = useAnalytics();
+  const {ticketInfoParams, fareContract, preassignedFareProduct} =
+    useTicketInfo(route.params.orderId);
+
+  const ticketType = route.params.ticketType;
 
   useApplePassPresentationSuppression();
 
+  const navigateToTicketInfoScreen = () => {
+    if (ticketInfoParams) {
+      analytics.logEvent(
+        'Ticketing',
+        'Ticket information button clicked',
+        ticketInfoParams,
+      );
+      navigation.navigate('Root_TicketInformationScreen', ticketInfoParams);
```

Comment:
It might not be part of this PR, but we extract `ticketInfoParams` from `useTicketInfo(route.params.orderId)` and send it to `Root_TicketInformationScreen` which inside finds the `preassignedFareProduct` that we already have as a result from `useTicketInfo(route.params.orderId)`. It would be better to just send the `TicketInfo` to the screens to avoid search the same object multiple times. I saw this patter on many other places, so might not require an action here.

Reply:
I remember it was a discussed in [this PR](https://github.com/AtB-AS/mittatb-app/pull/4127#discussion_r1427773994).

Basically the guideline for using navigation parameters is to pass ids, not entire objects. So I guess that is why this pattern is found in many places.

Reply:
That is correct, we send ids not objects. Because when using ids we get up-to-date data from the Firestore subscription, but if we start sending around objects then we risk that the data becomes outdated.